src/EditorFeatures/Core/LanguageServer/Handlers/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -53,7 +53,8 @@ public async Task<object[]> HandleRequestAsync(DocumentSymbolParams request, Req
             // TODO - Return more than 2 levels of symbols.
             // https://github.com/dotnet/roslyn/projects/45#card-20033869
             using var _ = ArrayBuilder<object>.GetInstance(out var symbols);
-            if (context.ClientCapabilities?.TextDocument?.DocumentSymbol?.HierarchicalDocumentSymbolSupport == true)
+            if (context.ClientCapabilities?.TextDocument?.DocumentSymbol?.HierarchicalDocumentSymbolSupport == true
+                || context.ServerKind == WellKnownLspServerKinds.AlwaysActiveVSLspServer)
```

Comment:
Could we add a comment for why we're doing this here? It seems odd that we're just outputting things even if the client capabilities say they don't support it? Why are the client capabilities not correct here?

Reply:
Ah, I see the other thread explained it -- but a comment here would still explain the hackery we'll have to do.

Reply:
Resolving as this code is outdated